app/services/namespaces/projects/flows/create_service.rb

```diff
@@ -36,14 +36,22 @@ def execute
               )
             end
 
-            flow = Flow.new(
+            flow = Flow.create(
```

Comment:
`create` already saves the flow in the database, so we get redundant queries with the `.save` call below
```suggestion
            flow = Flow.new(
```